migration-engine/connectors/sql-migration-connector/src/sql_migration_persistence.rs

```diff
@@ -183,26 +186,41 @@ async fn last_applied_migrations(
 }
 
 fn migration_table_setup_sqlite(t: &mut barrel::Table) {
-    migration_table_setup(t, types::date(), types::custom("TEXT"));
+    migration_table_setup(t, types::text(), types::date(), types::custom("TEXT"));
```

Comment:
I think this should be `datetime`, date will lose precision.

Reply:
Uh, this is SQLite and how things were before, do you want me to change them?

Reply:
I was pretty sure it was fixed a few weeks ago, maybe it was reverted by a careless PR. It should definitely be datetime, if it's not too hard to change.

Reply:
I can also change it in a follow-up PR, let's maybe do that.

Reply:
Nah, it was probably my merge conflict. Changing it back.

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/1108/commits/45e762cc5a4202d7838f9cd8a3677d9c74f5d59f